src/run/start.ts

```diff
@@ -14,10 +14,15 @@ export const startRun = async (client: ApiClient, config: ActionConfig): Promise
     overrideHostsByPool: config.run.overrideLoadGenerators
   });
 
-  const reportsUrl = webAppUrl(config, response.reportsPath);
-  const runsUrl = webAppUrl(config, response.runsPath);
-  return { runId: response.runId, reportsUrl, runsUrl };
+  // Fallback URLs for Gatling Enterprise Self-Hosted
+  const reportsPath = response.reportsPath ?? `/simulations/reports/${response.runId}`;
+  const runsPath = response.runsPath ?? `/simulations/runs/${config.run.simulationId}`;
```

Comment:
Copied from [here](https://github.com/gatling/frontline-ci-plugins/blob/7d14a34b47a6aee9a12e2cb7c3cf2903d52f92d2/frontline-ci-commons/src/main/java/io/gatling/frontline/ci/http/HttpClient.java#L57) and [here](https://github.com/gatling/frontline-ci-plugins/blob/7d14a34b47a6aee9a12e2cb7c3cf2903d52f92d2/frontline-ci-commons/src/main/java/io/gatling/frontline/ci/http/HttpClient.java#L70)